.python-version

```diff
@@ -1 +1 @@
-3.8.17
+3.9.17
```

Comment:
Bump to 3.9.18 if you have to retouch. This time I tested with that patch version.